src/main/java/org/maxgamer/quickshop/Event/ShopSuccessPurchaseEvent.java

```diff
@@ -36,14 +37,15 @@
 
     @Getter
     @NotNull
-    private final Shop shop;
+    public final Shop shop;
```

Comment:
In this case; the field must be private, and @Getter annotation will create getShop() method for it.

Reply:
okay but i don't understand how to use this getShop() method in the InternalListener.java